src/System.Net.Http/src/System/Net/Http/MessageProcessingHandler.cs

```diff
@@ -47,40 +47,46 @@ protected MessageProcessingHandler(HttpMessageHandler innerHandler)
 
                 // We schedule a continuation task once the inner handler completes in order to trigger the response
                 // processing method. ProcessResponse() is only called if the task wasn't canceled before.
-                sendAsyncTask.ContinueWithStandard(task =>
+                var tuple = Tuple.Create(this, tcs, cancellationToken);
```

Comment:
It may be long, but it's also important for type safety and catching things early.  If, for example, someone were to inadvertently change one of the arguments to Create, it would still compile fine, and it would only fail at run time if/when that object was cast inside of the continuation.  It be much better if such a change failed at compile time due to the types not matching up, and one way to get that is by not using var here.  So please follow the guidelines. Thanks.